test/verify/check-machines

```diff
@@ -379,7 +379,11 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         wait(lambda: "Initializing cgroup subsys" in self.machine.execute("cat {0}".format(args["logfile"])), delay=3)
         b.wait_in_text("#vm-subVmTest1-state", "Running")
 
-        # shut off
+        # shut off and wait for the spinner
+        b.click("#vm-subVmTest1-shutdown-button")
+        b.wait_visible("#vm-subVmTest1-shutdown-button.pf-m-in-progress.pf-m-disabled")
```

Comment:
Is this going to stay around forever? If not, this is super likely to be flaky test.

Reply:
It's not, but like in a real VM, when the user issues `shutdown` command, until it actually shuts down it really takes more time than necessary to spot it. It's the opposite for start up, therefore I did not check the spinner there.

Reply:
On the other hand, I feel it's something that we don't lose so much from not testing it. Maybe it's worth it to not introduce this kind of time based tests for not very important checks.

Reply:
Removed it.

Reply:
Thanks!
If there would be guarantee that shutdown takes at least n seconds I would be fine with it. But when it "usually takes longer" that feels like it is going to bite us sooner or later.